lib/cmd/fh3/projects/create.js

```diff
@@ -0,0 +1,61 @@
+/* globals i18n */
+var common = require("../../../common");
+var ini = require('../../../utils/ini');
+var templates = require('../../common/templates.js');
+var util = require('util');
+
+
+module.exports = {
+  'desc' : i18n._('Create project'),
+  'examples' :
+  [{
+    cmd : 'fhc projects create --title=<title>',
```

Comment:
@wtrocki tks for the review. However, the original command just works with the position as follows. ( the name in the desc was project-title either)

`````
Camilas-MacBook-Pro:fh-fhc cmacedo$ fhc projects create
(node) sys is deprecated. Use util instead.
fhc ERR! Invalid arguments
fhc ERR! Usage: 
fhc ERR! fhc projects [list] [<author-email>]
fhc ERR! fhc projects create <project-title> [<template-id>] --env=<environment>
fhc ERR! fhc projects update <project-id> <prop-name> <value>
fhc ERR! fhc projects read <project-id>
fhc ERR! fhc projects delete <project-id>
fhc ERR! fhc projects clone <project-id>
fh
````